src/Services/ActiveSubscriptions.ts

```diff
@@ -0,0 +1,41 @@
+import {DataRequestData, Request} from "../Request";
+import {HttpMethods, RequestTypes} from "../Constants";
+import {ITransactionResponse, ServiceList, TransactionResponse} from "../Models";
+export default class ActiveSubscriptions {
+    private readonly _serviceCode: string = 'GetActiveSubscriptions';
+
+    async get(){
+        return this.buildRequest().request().then((response) => {
+            return this.format(response.data);
+        });
+    }
+    private buildRequest(){
+        const data = new DataRequestData().setServiceList(new ServiceList({
+            name: this._serviceCode,
+            version: 1,
+            action: this._serviceCode,
+        }));
+        return new Request(RequestTypes.Data, HttpMethods.POST,data,TransactionResponse)
+    }
+    private format(data:ITransactionResponse){
+        let activeSubscriptions:IActiveSubscription[] = []
+        const xmlData =  data.services?.[0].parameters[0].value;
+        let parseString = require('xml2js').parseString;
+
+        if(typeof xmlData === 'string'){
+            parseString(xmlData,function (err:any, result:any) {
+                activeSubscriptions = result['ServiceCurrencies'].map((service:any) => {
```

Comment:
`result['ServiceCurrencies']` is causing an error. I believe it should be `result['ArrayOfServiceCurrencies']['ServiceCurrencies']` instead